### PR TITLE
Add class for attachment mime types

### LINKED ISSUE #613: Create MimeType constants for the Attachments object
#### Issue Summary

Here is an [example](https://github.com/sendgrid/sendgrid-php/blob/master/lib/mail/MimeType.php). The MimeTypes should be defined .in a new file /mail/AttachmentMimeType.php. Please use [this list](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types).

### PR DESCRIPTION
**Pull request by michaeljdennis**
# Resolves #613 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Add /lib/mail/AttachmentMimeType.php class that contains MIME type constants for attachments.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-php?pullRequest=721) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-php?pullRequest=721) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-php?pullRequest=721) it.</sub>

File: lib/mail/AttachmentMimeType.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+namespace SendGrid\Mail;
+
+/**
+ * This class is used to define the attachment mime types for the /mail/send API call
+ *
+ * @package SendGrid\Mail
+ */
+abstract class AttachmentMimeType
+{
+    const APPLICATION_ECMASCRIPT = "application/ecmascript";
```
Comment (peter279k) [inline lib/mail/AttachmentMimeType.php]: It seems that you want to collect the all available/important mime types?

It seems that some important mime types are not in this abstract class.

Please refer this [completed mime type lists ](https://www.sitepoint.com/mime-types-complete-list/) to complete this lists if possible.

**Comment by thinkingserious** [comment]
Hello @michaeljdennis,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by michaeljdennis** [comment]
Hi @thinkingserious,

I clicked the link to the form and it says "Your swag order has already been processed." without having filled out the form. Is that expected? Maybe it has my information saved from last year?

Thanks!

File: lib/mail/AttachmentMimeType.php (hunk 1 above)
Comment (michaeljdennis) [reply lib/mail/AttachmentMimeType.php]: @thinkingserious Which MIME types from the above list would you like added to this class?

**Comment by childish-sambino** [comment]
Closing as I don't see the value added.

**Pull request closed without merging**